### ISSUE TITLE
show doesn't work?

### ISSUE THREAD

**Issue by scidom** [JuliaStats/DataFrames.jl#778]
Not sure if this has been reported already, `show` is currently broken. If I try the simple example

```
df = DataFrame(A = 1:4, B = ["M", "F", "F", "M"])
```

from the manual, I get the error

```
Error showing value of type DataFrames.DataFrame:
ERROR: MethodError: `convert` has no method matching convert(::Type{Ptr{UInt8}}, ::Symbol)
This may have arisen from a call to the constructor Ptr{UInt8}(...),
since type constructors fall back to convert methods.
Closest candidates are:
  convert{T}(::Type{Ptr{T}}, ::UInt64)
  convert{T}(::Type{Ptr{T}}, ::Int64)
  convert{T}(::Type{Ptr{T}}, ::Ptr{T})
  ...

 in getmaxwidths at /home/theodore/.julia/v0.4/DataFrames/src/abstractdataframe/show.jl:109
 in show at /home/theodore/.julia/v0.4/DataFrames/src/abstractdataframe/show.jl:449
 in show at /home/theodore/.julia/v0.4/DataFrames/src/abstractdataframe/show.jl:437
 in anonymous at show.jl:1225
 in with_output_limit at ./show.jl:1202
 in showlimited at show.jl:1224
 in writemime at replutil.jl:2
 in display at REPL.jl:105
 in display at REPL.jl:108
 in display at multimedia.jl:149
 in print_response at REPL.jl:127
 in print_response at REPL.jl:112
 in anonymous at REPL.jl:588
 in anonymous at REPL.jl:790
 in anonymous at LineEdit.jl:760
 in prompt! at ./LineEdit.jl:1579
 in run_interface at ./LineEdit.jl:1547
 in run_frontend at ./REPL.jl:828
 in run_repl at ./REPL.jl:157
 in _start at ./client.jl:444
```

**Comment by garborg** [comment]
Thanks, @scidom -- I'll try to have a fix in today.

**Comment by spencerlyon2** [comment]
@garborg, any progress. I'm still seeing this today

**Comment by garborg** [comment]
The third of three upstream libraries that needed updating for 0.4 (not for this bug -- there are a few new 0.4 bugs on DataFrames master) will be tagged in METADATA in a few minutes. I've got a commit waiting that's good on 0.4 but probably needs a couple syntax tweaks to maintain 0.3 compatibility -- I'll try to get to it and tag a patch today.

**Comment by garborg** [comment]
Closed by #784. Thanks for the reports.

**Comment by scidom** [comment]
Thanks, it works now.
